lib/i18n/backend/simple.rb

```diff
@@ -10,14 +10,14 @@ module Backend
     # The implementation is provided by a Implementation module allowing to easily
     # extend Simple backend's behavior by including modules. E.g.:
     #
-    # module I18n::Backend::Pluralization
-    #   def pluralize(*args)
-    #     # extended pluralization logic
-    #     super
-    #   end
-    # end
-    #
-    # I18n::Backend::Simple.include(I18n::Backend::Pluralization)
+    #     module I18n::Backend::Pluralization
```

Comment:
I realize technically one space would suffice, but four seemed better for reading the docs inline so that the indentation is clearly intentional.  LMK if it should be done differently.

Reply:
I think this will work how you want it to :)